include/wasm/openmp_types.h

```diff
@@ -11,3 +11,6 @@ struct wasm_ident {
     I32 reserved_3;
     char const *psource;
 };
+
+// Types of source locations that call fork (defined in kmp.h)
+#define KMP_IDENT_WORK_LOOP 2
```

Comment:
This seem to be only used by OMPT (OpenMP tooling) in the original library, why is this necessary?

Reply:
I wanted to add some more checks in the fork call to make sure we weren't missing anything. I first wanted to check whether the source location information changed between `parallel for` and `parallel`, e.g. to indicate some different argument format. This flag seems to be defined alongside other potentially useful types of source location (https://github.com/llvm-mirror/openmp/blob/cb6f3c46a841e47718564ca40f49af8bcfc57c3a/runtime/src/kmp.h#L203) and I wanted to get an error if it ever changed. Turns out it doesn't, so this is probably useless.